content/en/dashboards/guide/custom_time_frames.md

```diff
@@ -3,28 +3,22 @@ title: Custom Time Frames
 kind: guide
 ---
 
+## Overview
+
+<div class="alert alert-info">Queries are run in UTC time, but the query time range is selected according to your <strong>browser</strong>. Toggle between displaying the default time zone or UTC from the dashboard configure action. For more information, see the <a href="/dashboards/#display-utc-time">Dashboard Settings documentation</a>.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Queries are run in UTC time, but the query time frame is selected according to your <strong>browser's time zone</strong>. Toggle between displaying the default time zone or UTC from the dashboard configure action. For more information, see the <a href="/dashboards/#display-utc-time">Dashboard Settings documentation</a>.</div>
```

- Time range -> time frame to be consistent with the rest of the page.
- browser -> browser's time zone (I'm assuming that's what it means?)